kolibri/core/content/management/commands/importcontent.py

```diff
@@ -71,8 +90,8 @@ def add_arguments(self, parser):
         parser.add_argument(
             "--node_ids",
             "-n",
-            # Split the comma separated string we get, into a list of strings
-            type=lambda x: x.split(","),
+            # Split the comma separated string we get, into a set of strings
+            type=lambda x: set(x.split(",")) if x else set(),
```

Comment:
Could we do this coersion to a set inside the `handle_async` method instead of here? Currently this introduces a different in behaviour between command line invocations of the management command and calls to `call_command` - as you can see below in the tests you didn't modify, `node_ids` and `exclude_node_ids` are still being set as lists. If the content manifest behaviour in particular is sensitive to this, then this may cause issues.

Reply:
Yeah, sure thing! In the newest commit, the argument is parsed into a list (an empty list if it is given an empty string), and we coerce it into a set inside `_transfer`.